pkg/chains/formats/intotoite6/testdata/taskrun2.json

```diff
@@ -8,6 +8,11 @@
         "serviceAccountName": "default"
     },
     "status": {
+        "annotations": {
+            "resolution.tekton.dev/path": "task.yaml",
+            "resolution.tekton.dev/revision": "123fdf35b4e7b1a56a84b2796aab2827edd65c25",
+            "resolution.tekton.dev/uri": "https://github.com/test/tekton-test.git"
+        },
```

Comment:
Fine for now, but highly recommend working towards making these actual fields in the Run status.